### PR TITLE
Dock(Badge) unread feature added in settings

### PR DESCRIPTION
**Pull request by fragm3**
Added enhancement, deck unread message toggle feature added to the setting page.
macOS Sierra Version: 10.12.5.
Issue: #192 
Type: Enhancement

off

on

### CODE REVIEW

File: app/renderer/js/pages/preference/general-section.js [hunk 1]
```diff
@@ -116,10 +128,27 @@ class GeneralSection extends BaseComponent {
 		$silentOption.addEventListener('click', () => {
 			const newValue = !ConfigUtil.getConfigItem('silent');
 			ConfigUtil.setConfigItem('silent', newValue);
+			ipcRenderer.send('toggle-dock-unread', newValue);
```
Comment (akashnimare) [inline app/renderer/js/pages/preference/general-section.js]: It should not be here.

**Review by akashnimare — CHANGES_REQUESTED** [review]

**Comment by akashnimare** [comment]
This needs a rebase and I'll review!

**Review by fragm3 — COMMENTED** [review]
Dock unread feature settings

**Comment by akashnimare** [comment]
Added in https://github.com/zulip/zulip-electron/commit/21f01d268a53edea2c4e0e5ba3aecb5bb63f1133.

**Pull request closed without merging**